vies/validators.py

```diff
@@ -16,7 +18,13 @@ def __init__(self, verify=True, validate=False):
         self.validate = validate
 
     def __call__(self, value):
+        if not isinstance(value, VATIN):
+            value = VATIN(*self.split_vatin(value))
         if self.verify:
             value.verify()
         if self.validate:
             value.validate()
+
+    def split_vatin(self, value):
```

Comment:
Can you maybe put this in a factory method on the VATIN class itself.
e.G. `VATIN.from_str('DE123456789')`